src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx

```diff
@@ -144,17 +157,19 @@ When building a decision, steps 1, 2 and 3 are optional on their own but at leas
 
     <tr>
       <td>
-        Step 3: Topology correlation
+        Step 3: Correlate topology (optional)
       </td>
 
       <td>
-        [Topology correlation](#topology) is currently in limited release. Contact your account representative to enable this feature. Once enabled, you'll first need to ingest your topology data via our NerdGraph `aiTopologyCollector` (search for aiTopology in the [NerdGraph GraphiQL explorer](https://api.newrelic.com/graphiql)). This allows any topology-related decision to be matched with your topology data. [Learn more about setting up topology correlation](#topology).
+        For automatic topology correlation, make sure your telemetry data is collected by [New Relic agents](/docs/new-relic-solutions/new-relic-one/install-configure/compatibility-requirements-new-relic-agents-products/). Learn more about [topology correlation out-of-the-box]([#topology-requirements]).
```

Comment:
The "topology-requirements" link gives an error. I think we need to remove the [] inside the ().

Reply:
@barbnewrelic, good catch! I just fixed this!